### ISSUE TITLE
[1.12.2] vampirism&perfomant crash

### ISSUE THREAD

**Issue by nightloli** [ldtteam/Performant#15]
**Describe the bug**
Crash with mod vampirism on capturing village

**To Reproduce**
1) Install vampirism and perfomant on minecraft 1.12.2
2) Become vampire with /vampirism level Vampire anyNumber
3) Find a village
4) Capture it by clicking on totem looking kinda like a beam and killing all angry villagers, that will be trying to attack you
5) Get crash

**Expected behavior**
Expected to get captured village

**Screenshots**
minecraft just closes

** Mod version **
performant 1.12.2-1.8
vampirism  1.6.2
VampirismIntegrations-1.12.2-1.2.0.jar  

 |
crash-report:
https://pastebin.com/FGaZfNR3
latest log:
https://pastebin.com/UVN5RqcT

**Comment by maxanier** [comment]
```
        List<EntityCreature> list = this.world.getEntitiesWithinAABB(EntityCreature.class, bb;
```
Will return all entities and not just EntityCreatures with this mod installed.
This causes Vampirism to crash

**Comment by someaddons** [comment]
fixed in 1.9
